.github/workflows/mpet-docker-test.yml

```diff
@@ -0,0 +1,65 @@
+name: MPET docker CI
+
+on:
+  pull_request:
+    branches: [ master,development ]
+
+jobs:
+  test:
+    
+    runs-on: ubuntu-latest
+    container:
+      image: vikko00/mpet-deps:latest
+
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 1
+        path: mpet
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 1
+        path: base-mpet
+        ref: ${{ github.base_ref }}
```

Comment:
It seems like the test or CI should run against the proposed changes and not the base branch?

Reply:
The results of the proposed changes are compared to the results of the base branch. Therefore I think it would be easiest to run the tests for the base branch as well.